addons/l10n_ar/data/account_tax_group.xml

```diff
@@ -63,7 +63,6 @@
         <field name="name">Otros Impuestos</field>
         <field name="sequence">20</field>
         <field name="l10n_ar_tribute_afip_code">99</field>
-        <field name="country_id" ref="base.ar"/>
```

Comment:
only once?

Reply:
Are you asking if it's the only record doing this? If yes, I believe so.